docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -74,6 +74,21 @@ image::images/alert-flyout-add-action.png[You can add multiple actions on an ale
 Actions are not required on alerts. In some cases you may want to run an alert without actions first to understand its behavior, and configure actions later.
 ==============================================
 
+[float]
+=== Global actions configuration
+Some actions configuration options apply to all actions.
+If you are using an *on-premises* Elastic Stack deployment, you can set these in the kibana.yml file.
+If you are using a cloud deployment, you can set these via the console.
+
+Here's a list of the available global configuration options and an explanation of what each one does:
+
+* `xpack.actions.allowedHosts`: Set to an array of host names which actions such as email, slack, pagerduty, and webhook can connect to. An element of * indicates any host can be connected to. An empty array indicates no hosts can be connected to. Default: [ {asterisk} ]
+* `xpack.actions.enabledActionTypes`: Set to an array of action types that are enabled. An element of * indicates all action types registered are enabled. The action types provided by Kibana are: .server-log, .slack, .email, .index, .pagerduty, .webhook. Default: [ {asterisk} ]
+* `xpack.actions.proxyUrl`: Specifies the proxy URL to use, if using a proxy for actions.
```

Comment:
For this, and the next one, how about using "Specify" so that these items start the same way as other items in the list.

Reply:
Makes sense.